src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -497,7 +505,7 @@ private ProgrammingExerciseStudentParticipation copyBuildPlan(ProgrammingExercis
         if (!participation.getInitializationState().hasCompletedState(InitializationState.BUILD_PLAN_COPIED) || participation.getBuildPlanId() == null) {
             final var projectKey = participation.getProgrammingExercise().getProjectKey();
             final var planName = BuildPlanType.TEMPLATE.getName();
-            final var username = participation.getStudent().getLogin();
+            final var username = participation.getParticipantIdentifier();
```

Comment:
rename username to participantIdentifier

Reply:
Renamed